File: GoChat/src/main/java/com/jw/gochat/fragment/MeFra.java
package com.jw.gochat.fragment;

import android.view.View;

import com.jw.gochat.R;
import com.jw.gochat.base.BaseFragment;

/**
 * Created by Administrator on 2017/3/26.
 */


public class MeFra extends BaseFragment {

    @Override
    public View bindView() {
        View view=View.inflate(getActivity(),R.layout.fragment_me,null);
        return view;
    }

}

File: goChat/src/main/java/com/jw/gochat/fragment/MeFra.kt
package com.jw.gochat.fragment

import android.os.Bundle
import com.jw.gochat.R
import com.jw.gochat.databinding.FragmentMeBinding
import com.sencent.mm.GoChatBindingFragment

/**
 * Created by Administrator on 2017/3/26.
 */


class MeFra : GoChatBindingFragment<FragmentMeBinding>() {

    override fun getLayoutId() = R.layout.fragment_me

    override fun doConfig(arguments: Bundle?) {
    }
}